modules/clientBinaryManager.js

```diff
@@ -180,7 +180,9 @@ class Manager extends EventEmitter {
         .then((localConfig) => {
             if (!localConfig) {
                 log.info('No config for the ClientBinaryManager could be loaded, using local clientBinaries.json.');
-                localConfig = require('../clientBinaries.json');
+
+                const localConfigPath = path.join(Settings.userDataPath, 'clientBinaries.json');
```

Comment:
Yes, that's right. The next line will simply return and jump to the next block. As the `localConfig` is empty it will use my new code to fetch the already downloaded. If that one is not available (first run of Mist is offline) it uses the bundled. Those configs are both validated and so this plays nice with the omiting of the validation block.